lms/validation/_base.py

```diff
@@ -68,13 +68,13 @@ class MySchema(PyramidRequestSchema):
 
     def __init__(self, request):
         super().__init__()
-        self.context = {"request": request}
+        self._context = {"request": request}
```

Comment:
See https://hypothes-is.slack.com/archives/C1MA4E9B9/p1738930836166139?thread_ts=1738678637.328809&cid=C1MA4E9B9